homeassistant/components/smhi/const.py

```diff
@@ -10,5 +10,6 @@
 DOMAIN = "smhi"
 
 HOME_LOCATION_NAME = "Home"
+DEFAULT_NAME = "Weather"
 
-ENTITY_ID_SENSOR_FORMAT = WEATHER_DOMAIN + ".smhi_{}"
+ENTITY_ID_SENSOR_FORMAT = Platform.WEATHER + ".smhi_{}"
```

Comment:
We are mixing concepts here a bit.  I'd probably leave this as-is, but let Home Assistant pick the entity anyways

Reply:
Ok, no problem to revert this. Just thought it would look a bit cleaner also here to put the Platform enum.